core/biogeophys/cable_common.F90

```diff
@@ -24,13 +24,22 @@
 
 MODULE cable_common_module
 
+  USE NetCDF,   only: NF90_INQ_DIMID, NF90_NOERR, NF90_BADDIM
   IMPLICIT NONE
 
   !---allows reference to "gl"obal timestep in run (from atm_step)
   !---total number of timesteps, and processing node
   INTEGER, SAVE :: ktau_gl, kend_gl, knode_gl, kwidth_gl
   INTEGER, SAVE :: CurYear  ! current year of multiannual run
 
+  ! Dimension name lists for searching NetCDF dimensions IDs
+  CHARACTER(LEN=16), DIMENSION(5) :: LatNames = &
+    (/"latitude", "lat", "lats", "y", "Latitude"/)
```

Comment:
You can use square brackets (Using `(/ x /)` is part of the old standard in Fortran)

Reply:
Also, it'd be great if you could define it as a parameter / constant (so that it is faster in runtime).

Reply:
See comment about [derived types](https://github.com/CABLE-LSM/CABLE/pull/377#discussion_r1760568177) earlier, which would contradict introducing `PARAMETER`. Happy for you to implement any

Reply:
Is there a good reason to use square brackets over the old standard? Or has Fortran updated their standards?

Reply:
So I noticed that the current fortran documentation has all `[]` braces. Also, although it may be subjective, the author of the book Modern Fortran preferred to use square braces as well. That makes using `[]` optional, but I used it in `cable_checks`,  so it'd be nice to have a common coding standard.

![image](https://github.com/user-attachments/assets/dcf37ec0-2119-445a-bd1b-6e1c0fa711a4)

Reply:
That's a pretty good reason, I'll make that change.